.github/workflows/preview.yml

```diff
@@ -3,43 +3,91 @@ name: Publish Preview release
 on:
   schedule:
     # run every day at 00:15 UTC to avoid high load times at the start of every hour
-    - cron: '15 0 * * *'
+    - cron: "15 0 * * *"
 
 jobs:
-  publish:
-    name: Publish Preview to VS Code Marketplace
-    runs-on: ubuntu-latest
+  build:
+    name: Package
+    strategy:
+      matrix:
+        include:
+          - os: windows-latest
+            platform: win32
```

Comment:
What are `platform` and `arch` doing? I didn't see any references to them, and they shouldn't affect the [runner platform](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on) as `runs-on` only uses `target`.

Reply:
This approach was taken from the [platform specific extensions CI example](https://github.com/microsoft/vscode-platform-specific-sample/blob/main/.github/workflows/ci.yml#L61). Things like `platform` and `arch` were supposed to be combined inside the shell commands to make environment variables like `"target=${{ matrix.platform }}-${{ matrix.arch }}`.

However, if I did that I would have to write out different environment variable statements for different shells, since each OS has a different way to do that. If I used powershell, it could be one command for all, but we don't do that here.

I chose instead to be more verbose in the matrix and add `target` which has the correctly combination of platform and arch. We technically can remove `platform` and `arch`, since they're not used at the moment.